### PR TITLE
[DQM] Updated the client for MuonMonitor in online DQM

### PR DESCRIPTION
**Pull request by calderona**
#### PR description:

There were some missing piece in the muon monitor client for the muon DQM that crashed at the online DQM. Now it has been fixed, and it is expected to be included on next MWGR.

#### if this PR is a backport please specify the original PR:

https://github.com/cms-sw/cmssw/pull/28488

### CODE REVIEW

**Comment by fioriNTU** [comment]
please test

**Comment by jfernan2** [comment]
+1

File: DQM/Integration/python/clients/mutracking_dqm_sourceclient-live_cfg.py [hunk 1]
```diff
@@ -96,16 +96,35 @@
                                    process.muonCSCDigis +
                                    process.muonDTDigis +
                                    process.muonRPCDigis +
-                                   process.muonRPCNewDigis +
```
Comment (silviodonato) [inline DQM/Integration/python/clients/mutracking_dqm_sourceclient-live_cfg.py]: Comparing this backport with the original PR #28488, I see you are removing muonRPCNewDigis from this sequence. It was already removed in 11_1_X. 
See: https://github.com/cms-sw/cmssw/pull/28488/files#diff-b25c3200c94b7e485b033fb90ab6e51fL99 .
What are the effects of this change?

File: DQM/MuonMonitor/test/mutracking_dqm_sourceclient-live_cfg.py [hunk 2]
```diff
@@ -136,15 +136,30 @@
                                    process.muonCSCDigis +
                                    process.muonDTDigis +
                                    process.muonRPCDigis +
-                                   process.muonRPCNewDigis +
```
Comment (silviodonato) [inline DQM/MuonMonitor/test/mutracking_dqm_sourceclient-live_cfg.py]: same here
See: https://github.com/cms-sw/cmssw/pull/28488/files#diff-0298319a228a66632d6d427ae8910e8dL139

**Comment by silviodonato** [comment]
please test

**Comment by silviodonato** [comment]
+1

**Pull request merged**